scripts/objectiveCompletionResponder.js

```diff
@@ -0,0 +1,60 @@
+function onObjectiveCompleted(self, event, world) {
```

Comment:
This file is logic copied from `twilioquest-base` as per discussion with Kevin in backlog grooming 4/26/2021. We don't love this re-use, because it's fragile and won't benefit from updates. However, a more robust solution was out of scope for this PR and ticket.